README.md

```diff
@@ -0,0 +1,54 @@
+ # AutoQUBO
```

Comment:
We are a bit worried about trademarks, and will change AutoQUBO to AUTOmated QUBO Generator in the readme. We can leave it as is in the paper and the url.